yaml/kubernetes/security/exposing-docker-socket-hostpath.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+- id: exposing-docker-socket-hostpath
+  patterns:
+  - pattern-inside: |
+      volumes:
+        ...
+  - pattern: |
+      hostPath:
+        ...
+        path: /var/run/docker.sock
+  message: |
+    Container is running with a writable root filesystem. This may
+    allow malicious applications to download and run additional payloads, or
+    modify container files. If an application inside a container has to save
+    something temporarily consider using a tmpfs. Add 'readOnlyRootFilesystem: true'
+    to this container to prevent this.
```

Comment:
Is the message supposed to mention docker sockets instead? 🤔

Reply:
Oops, yeah this is totally the wrong message. I'll update :+1: